### PR TITLE
ESR Short Descriptions Field Validation

### PR DESCRIPTION
**Pull request by sarwarahmed1**
Reviewed

### CODE REVIEW

File: src/test/java/ServiceNow/ESR/Playwright/Features/ESRShortDescription.feature [hunk 1]
```diff
@@ -13,4 +13,18 @@ Feature: ESR Short Description Fields
...
+    Then user confirms the following fields: ESR ID, Stage, Short Description and Project Name
+
+  @ESR-527 @sarwarahmed1 @Progression @playwright
+  Scenario: Test the listed short descriptions fields in an ESR-Q ticket
+    Given a user logs into Native View on the NCI at your service page
+    And navigates to create ESR
+    And fills out all required information for ESR-Q and clicks submit
+    Then user confirms the following fields: Number, Approval, State and Requested For
+
+  @ESR-528 @sarwarahmed1 @Progression @playwright
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/ESR/Playwright/Features/ESRShortDescription.feature:25]: Per Diego, please tag these scenarios @In-Progress if you are still working on them

**Review by hamidnesar — CHANGES_REQUESTED** [review]
Hi, please review the comment

**Review by Mariachaudhry — APPROVED** [review]

File: src/test/java/ServiceNow/ESR/Playwright/Features/ESRShortDescription.feature (hunk 1 above)
Comment (sarwarahmed1) [reply src/test/java/ServiceNow/ESR/Playwright/Features/ESRShortDescription.feature:25]: Forgot to change them, should be regression tags

**Review by hamidnesar — APPROVED** [review]
Reviewed and approved!

**Review by katm007 — APPROVED** [review]
approved!

**Pull request merged**